backend/base/templates/org_preview2.html

```diff
@@ -1,688 +1,848 @@
-                                        <p>B. YOU UNDERSTAND AND AGREE THAT UNDER NO CIRCUMSTANCES, INCLUDING, BUT NOT LIMITED TO, NEGLIGENCE, SHALL WE BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, PUNITIVE, EXEMPLARY, OR CONSEQUENTIAL DAMAGES INCLUDING, BUT NOT LIMITED TO, DAMAGES FOR LOSS OF USE, DATA, OR PROFITS, ARISING OUT OF THE USE OR THE INABILITY TO USE THE SITE, WEBSITES LINKED TO THE SITE, THE CONTENT OR THE SERVICES CONTAINED IN OR ACCESSED THROUGH THE SITE, OR ANY SERVICES OR PRODUCTS RECEIVED THROUGH THE SITE. YOU SPECIFICALLY ACKNOWLEDGE AND AGREE THAT WE ARE NOT LIABLE FOR ANY CONDUCT OF ANY USER.</p>
-                                        <p>6. INDEMNIFICATION You agree to indemnify and hold us, our parents, subsidiaries, affiliates, officers and employees, harmless from any claim, demand, or damage, including reasonable attorneys' fees, asserted by any third party due to or arising out of your use of or conduct on the Site.</p>
-                                        <p>7. GENERAL You agree that all matters relating to your access to or use of the Site shall be governed by the laws of the State of California. You agree and hereby submit to the exclusive personal jurisdiction and venue any court of competent jurisdiction within the State of California with respect to such matters.</p>
+                                    </form>
+                                </div>
+                                <!-- Terms and Conditions Modal -->
+                                <div class="modal fade" id="termsModal" tabindex="-1" aria-labelledby="termsModalLabel"
+                                    aria-hidden="true">
+                                    <div class="modal-dialog modal-lg">
+                                        <div class="modal-content">
+                                            <div class="modal-header">
+                                                <h5 class="modal-title" id="termsModalLabel">Terms and Conditions</h5>
+                                                <button type="button" class="btn-close" data-bs-dismiss="modal"
+                                                    aria-label="Close"></button>
+                                            </div>
+                                            <div class="modal-body">
+                                                <!-- Here you can include the terms and conditions text -->
+                                                <p><strong>General Site Usage</strong></p>
+                                                <p>Last Revised: December 16, 2013</p>
+                                                <p>Welcome to www.lorem-ipsum.info. This site is provided as a service to our

```

Comment:
Is this dummy texts?

Reply:
Yes. Till client decide the Terms. Got some from online.